cypress/integration/participantStatus.spec.js

```diff
@@ -140,7 +165,7 @@ describe('Participants status suite', () => {
   };
 
   it('Flow test: available --> engage by HA --> hired --> archived & acknowledged', () => {
-    const participantId = 4;
+    const participantId = 5;
```

Comment:
This is due to the new test data?

Reply:
No, I changed it during local testing as that participant was already used in my local env. I'll revert